src/lib/y2storage/md.rb

```diff
@@ -288,7 +288,11 @@ def md_users
     end
 
     def types_for_is
-      super << :md
+      types = super
+      types << :md
+      types << :raid if software_defined?
```

Comment:
I guess this `if software_defined?` is here to exclude MD containers (in combination with the `types << :raid` on `MdMember` below). It definitely needs some comment because it's a really tricky implementation.